lib/new_relic/telemetry_sdk/clients/client.rb

```diff
@@ -26,9 +26,18 @@ def initialize host:,
         add_user_agent_header @headers
         add_content_encoding_header @headers if @gzip_request
         @connection_attempts = 0
-        @max_retries= 8 # based on config
-        @backoff_factor = 5 # based on config
-        @backoff_max = 80 # based on config
+      end
+
+      def max_retries
+        TelemetrySdk.config.max_retries
+      end
+
+      def backoff_factor
+        TelemetrySdk.config.backoff_factor
+      end
+
+      def backoff_max
+        TelemetrySdk.config.backoff_max
```

Comment:
Should these be private? Now that have rebased on dev and there is a private section, would it make sense to move these into there? I'm not sure it's necessary but i'm not sure.

Reply:
Made these private.  good call out.  but perhaps we can chat more about the reasoning behind making things public/private?